app/controllers/metadata_controller.rb

```diff
@@ -0,0 +1,25 @@
+class MetadataController < ApplicationController
+  layout Proc.new { |controller| controller.request.xhr? ? false : nil }, :only => :upload
```

Comment:
@benwbrum I wanted to suppress the layout for the `upload` action, I was not sure it would be needed for the other actions.

But it's clear that we don't need it now, I changed it to `layout false` as you suggested, thanks.